db/installations.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hi Joel - I can't add a comment to the file because it is in JSON format 😢 . But this file stores the `installation` object for each team (without the tokens)

Reply:
I forgot about that 😄 Could you add a comment in the `helpers.js` file please - above the usage of `storeLocalInstallation`?  It isn't clear what it is for, and once this PR is closed, nobody will see this thread.